Add naive instructions to README.md for building and running.

apetro opened the pull request:
These instructions result in an app at least launching.
Looks like there's more to be done to make the launched app interesting.

apetro committed:
Add naive instructions to README.md for building and running.

These instructions result in an app at least launching.
Looks like there's more to be done to make the launched app interesting.

ChristianMurphy commented on README.md:
````diff
@@ -1,3 +1,19 @@
+
+Build using 
+
+```shell_session
````
Github doesn't recognize recognize `shell_session` as a language, these are the aliases that will trigger code highlights. https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L3953-L3957

apetro replied:
Ah. I was relying upon [the Rouge lexers](https://github.com/jneen/rouge/wiki/List-of-supported-languages-and-lexers) [supported by GitHub Pages](https://help.github.com/articles/using-syntax-highlighting-on-github-pages/), hadn't realized it's a different set for GitHub itself.

ChristianMurphy replied:
Interesting, I'm surprised linguist doesn't support all the same aliases as Rouge, since the two are supposed to be interchangeable. :confused:  :man_shrugging:

ChristianMurphy reviewed (approved):
LGTM, Thanks @apetro!

apetro committed:
Highlight as `ShellSession` not `shell_session`

Use the GitHub-supported `linguist` [name for shell sessions](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L3986) rather than the GitHub-Pages-supported `Rouge` name for shell sessions in marking shell example for code highlighting.

In practice currently a no-op, since neither `shell_session` nor `ShellSession` (nor `shell`) yield interesting highlighting for this specific code block. Nonetheless, the code highlighting language type declaration probably adds a tiny bit of value in clarifying what the example in the code block is.

apetro commented:
This instructions will no longer be correct when #2 merges. That's okay. #2 is progress. But will need to rev these instructions, or close PR and try again in light of the `onejar` approach of #2.

BKKirchoff commented:
Hi Andrew,

I am going to need you to translate these messages into non technical language for me until I learn the lingo. I am lost right now.

Best wishes,

Bruce
-----------
Bruce Kirchoff
"Education is that which remains, if one has forgotten everything he learned in school."
- Einstein

On Feb 6, 2017, 10:27 AM -0500, Andrew Petro <notifications@github.com>, wrote:
>
> This instructions will no longer be correct when #2 (https://github.com/Jasig/ImageQuiz/pull/2) merges. That's okay. #2 (https://github.com/Jasig/ImageQuiz/pull/2) is progress. But will need to rev these instructions, or close PR and try again in light of the onejar approach of #2 (https://github.com/Jasig/ImageQuiz/pull/2).
>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub (https://github.com/Jasig/ImageQuiz/pull/5#issuecomment-277716088), or mute the thread (https://github.com/notifications/unsubscribe-auth/ARqa-a-x9lgoz4j6JuM_0q7zQxx6dD9Mks5rZzvQgaJpZM4L0uQB).
>
>
>
